data/ui/window.ui

```diff
@@ -9,7 +9,7 @@
       <object class="AdwToastOverlay" id="toast_overlay">
 
         <property name="child">
-          <object class="GtkBox">
+          <object class="GtkBox" id="window_box">
```

Comment:
I believe you should change the child GtkBox and not this one

Reply:
Fixed now.